css/build.css

```diff
@@ -34,27 +34,48 @@
   cursor: pointer;
 }
 
+[data-has-notch].platform-web .dynamic-list-add-item {
+  bottom: calc(15px + 34px); /* For Fliplet Studio */
+}
```

Comment:
Currently `[data-has-notch]` would only apply to `.platform-web` anyway, but leave it is. It won't be a problem and I don't know how this might evolve just yet.